src/status_im/contexts/chat/messenger/messages/content/status_link_preview/view.cljs

```diff
@@ -9,20 +9,21 @@
         link-previews?       (rf/sub [:chats/message-link-previews? chat-id message-id])]
     (when (seq status-link-previews)
       [rn/view {:style {:margin-top (when link-previews? 8)}}
-       (for [{:keys [url community]} status-link-previews]
-         (when community
-           (let [{community-description :description
-                  community-icon        :icon
-                  community-banner      :banner
-                  community-name        :display-name
-                  members-count         :members-count} community]
-             ^{:key url}
-             [quo/internal-link-card
-              {:type          :community
-               :size          :message
-               :description   community-description
-               :members-count members-count
-               :title         community-name
-               :banner        (:url community-banner)
-               :icon          (:url community-icon)
-               :on-press      #(rf/dispatch [:universal-links/handle-url url])}])))])))
+       (doall
```

Comment:
@ibrkhalil I don't understand why this `doall` is needed. The body of `for` doesn't have any side-effects, such as `deref`. I couldn't reproduce the warning. Could you share reproduction steps to see the warning from Reagent?

I used this snippet to check:

```diff
modified   src/status_im/contexts/chat/messenger/messages/content/status_link_preview/view.cljs
@@ -9,21 +9,20 @@
         link-previews?       (rf/sub [:chats/message-link-previews? chat-id message-id])]
     (when (seq status-link-previews)
       [rn/view {:style {:margin-top (when link-previews? 8)}}
-       (doall
-        (for [{:keys [url community]} status-link-previews]
-          (when community
-            (let [{community-description :description
-                   community-icon        :icon
-                   community-banner      :banner
-                   community-name        :display-name
-                   members-count         :members-count} community]
-              ^{:key url}
-              [quo/internal-link-card
-               {:type          :community
-                :size          :message
-                :description   community-description
-                :members-count members-count
-                :title         community-name
-                :banner        (:url community-banner)
-                :icon          (:url community-icon)
-                :on-press      #(rf/dispatch [:universal-links/handle-url url])}]))))])))
+       (for [{:keys [url community]} status-link-previews]
+         (when community
+           (let [{community-description :description
+                  community-icon        :icon
+                  community-banner      :banner
+                  community-name        :display-name
+                  members-count         :members-count} community]
+             ^{:key url}
+             [quo/internal-link-card
+              {:type          :community
+               :size          :message
+               :description   community-description
+               :members-count members-count
+               :title         community-name
+               :banner        (:url community-banner)
+               :icon          (:url community-icon)
+               :on-press      #(rf/dispatch [:universal-links/handle-url url])}])))])))
modified   src/status_im/subs/messages.cljs
@@ -115,7 +115,16 @@
  :chats/message-status-link-previews
  :<- [:messages/messages]
  (fn [messages [_ chat-id message-id]]
-   (get-in messages [chat-id message-id :status-link-previews])))
+   (let [all (get-in messages [chat-id message-id :status-link-previews])]
+     (concat all
+             [(update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))
+              (update (first all) :url str (rand-int 1000000000))]))))
```